docs/indepth/installation.rst

```diff
@@ -18,8 +18,17 @@ It contains the services that need to run on each node.
 Installing ``flocker-cli``
 ==========================
 
-Linux
------
+Ubuntu 14.04
```

Comment:
The Getting Started Guide has these instructions under "Ubuntu / Debian" - this has them under "Ubuntu 14.04". Have these instructions been tested on Debian? Are they version specific? This makes me think that a Debian user might try the "Other Linux Distributions" option here and the "Ubuntu / Debian" instructions in the GSG. Note that none of this is your fault, it is just highlighted by this branch. Change for consistency or create a follow-up.